Overview.src.html

```diff
@@ -3979,9 +3977,51 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 is itself associated with <span title=concept-Response-response>response</span>'s
 <span title=concept-response-header-list>header list</span>.
 
-<p>A <code>Response</code> object's <span title=concept-Body-body>body</span> is its
-<span title=concept-Response-response>response</span>'s
-<span title=concept-response-body>body</span>.
+<p>A <code>Response</code> object also has an associated
+<dfn title=concept-Response-readable-stream>readable stream</dfn> which is null or a
+<span title=concept-ReadableStream>ReadableStream</span>. Its initial value is null.
```

Comment:
ReadableStream is an object, right? Should we not use `<code>` for it and follow it with the word object? E.g., `<code title=...>ReadableStream</code> object`?